docs/architecture/specs/nft/README.md

```diff
@@ -0,0 +1,291 @@
+   * [Links](#links)
+
+
+---
+
+This SPEC introduces Non Fungible Tokens (NFTs) as part of Nevermined architecture allowing the tokenization of assets, sales with secondary market royalties support and exclusive access to NFT holders.
+
+## Motivation
+
+With this Specification we want to add to Nevermined the technical capabilities allowing the users to have a platform where digital and physical assets can be represented and traded between people openly, but keeping security, integrity, provenance and content creators attribution.
+
+With that objective in mind, Nevermined must facilitate the execution of use cases like the following:
+
+* Art publishing, exhibition, selling. Secondary market
+* Collection of objects
+* Tokenization of digital and physical assets (real estate, etc.)
+* etc
+
+The intention of this document is to discuss how a fully functional NFT engine can be established in a decentralized system, to allow to support complex use cases related with assets tokenization.
+
+The main objectives of this SPEC are:
+
+* Understand the main use cases we can to implement
+* Understand how assets can be tokenized using NFTs
+* Understand how asset and NFTs sales can be established between creators, owners and consumers
+* Identify how NFTs rely in some other Nevermined building blocks
+* Specify how NFTs associated to assets (DIDs) can be minted and burner
+* Allow the definition of royalties that enable the retribution of original creators in the secondary market
+* Specify how NFT holders can get access to existing services
+* Understand what provenance information needs to be tracked
+
+
+## Use Case
+
+The following use case helps to understand the scope of the problem driving the technical implementation to put in place.
+For this use case we have take into account the following target users:
+
+* Publishers (an artists for example)
+* Consumers (a collector)
+* Facilitators (an art gallery for example)
+
+### Publisher
+
+As a **Publisher**:
+
+* I want to register digitally an asset
+* I want to associate some metadata information to the assets I register
+* I want to show my track record as publisher (bio, what I created, what I sold, etc.)
+* I want to sell a digital representation of an asset for some price
+* I want to limit the number of copies of an asset for sale
+* I want to be rewarded via royalties in the secondary market for further sales
+* I want to be able to delegate or loan some of my assets to a **Facilitator** user (gallery, etc.)
+
+
+### Consumer
+
+As a **Consumer**:
+
+* I want to discover relevant assets filtering by multiple parameters
+* I appreciate curated assets avoiding me to waste time searching for high quality
+* I want to see a digital representation of an asset and detailed information of it
+* I want to check the ownership of an asset and full provenance record
+* I want to see how many pieces of an specific assets were created
+* I want to purchase an asset or part of an asset series
```

Comment:
```suggestion
* I want to purchase an asset or an asset that is part of an asset series
```

Reply:
The model I have in my head is an Asset can be tokenized (via NFTs) for each division of an Asset is not a Nevermined Asset itself (meaning a NFT is not a DID). That would require some bigger changes.
Partial ownership is not something yet specified but would be nice to explore, maybe as part of a different Spec. I was thinking probably makes sense to use multisig wallets for that.